lib/lhm/sql_retry.rb

```diff
@@ -39,6 +39,8 @@ def default_retry_config
             /Lock wait timeout exceeded/,
             /Timeout waiting for a response from the last query/,
             /Deadlock found when trying to get lock/,
+            /Query execution was interrupted/,
+            /Lost connection to MySQL server during query/,
           ]
```

Comment:
Maybe should also handle this one that we see recently, too?

```suggestion
            /Max connect timeout reached/,
          ]
```

Reply:
Oh I must have missed it in those screenshots. Updated.